narwhals/stable/v1/__init__.py

```diff
@@ -1175,6 +1175,10 @@ def concat_str(
     )
 
 
+def format(f_string: str, *args: IntoExpr) -> Expr:
+    return _stableify(nw.format(f_string, *args))
```

Comment:
Should this have the minimal docstring?

```suggestion
def format(f_string: str, *args: IntoExpr) -> Expr:
    """Format expressions as a string."""
    return _stableify(nw.format(f_string, *args))
```

Reply:
we don't have these anywhere else in v1